package.json

```diff
@@ -54,17 +54,18 @@
     "expo-updates": "*"
   },
   "devDependencies": {
-    "@types/mkdirp": "^1.0.0",
+    "@types/mkdirp": "^1.0.2",
     "@types/node": "^14.0.1",
-    "@types/react-native": "^0.64.10",
-    "@types/rimraf": "^3.0.0",
-    "expo-application": "~3.2.0",
-    "expo-constants": "~11.0.0",
-    "expo-device": "~3.3.0",
+    "@types/react-native": "^0.68.2",
+    "@types/rimraf": "^3.0.2",
+    "expo-application": "~4.1.0",
+    "expo-constants": "~13.1.1",
+    "expo-device": "~4.2.0",
     "expo-module-scripts": "^2.0.0",
-    "expo-updates": "~0.7.0",
-    "react": "16.13.1",
-    "react-native": "^0.64.1",
-    "react-native-web": "~0.13.7"
+    "expo-updates": "~0.13.3",
+    "react": "17.0.2",
+    "react-native": "^0.68.2",
+    "react-native-web": "~0.17.7",
```

Comment:
@brentvatne Should these dependency versions match SDK 46 if it's released as part of that?